### PR TITLE
Output remote state information

### PR DESCRIPTION
**Pull request by tnation14**
Export information relevant to ECS tasks for use with other projects, including:
- VPC id
- Container instance security group, instance name, and IAM role
- Public hosted zone.

Also, bump terraform version to 0.9.5 and update the path to the remote state and tfvars.

# Testing
See geotrellis/geotrellis-site#62

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ addons:
 
 env:
   global:
-    - TERRAFORM_VERSION=0.9.3
+    - TERRAFORM_VERSION=0.9.5
```
Comment (hectcastro) [inline .travis.yml]: Any reason not to go to 0.9.6 here?

File: terraform/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+output "vpc_id" {
+  value = "${module.vpc.id}"
+}
+
+output "logs_bucket_id" {
+  value = "${aws_s3_bucket.logs.id}"
```
Comment (hectcastro) [inline terraform/outputs.tf:6]: Just to be clear, outputs are not necessary for top level resources like these, but you're doing it here for consistency?

Comment (tnation14) [reply terraform/outputs.tf:6]: As far as I understand, the output *is* necessary; if I don't export this value here, I don't think it's accessible when using the `terraform_remote_state` data source elsewhere. See [hashicorp/terraform#11932](https://github.com/hashicorp/terraform/issues/11932#issuecomment-298719332)

File: scripts/infra [hunk 3]
```diff
@@ -24,13 +24,17 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [[ -n "${GT_SITE_SETTINGS_BUCKET}" ]]; then
         pushd "${DIR}/../terraform"
 
+        aws s3 cp "s3://${GT_SITE_SETTINGS_BUCKET}/terraform/core/terraform.tfvars" \
```
Comment (hectcastro) [inline scripts/infra:27]: We should add the corresponding `rm` to `.travis.yml` in a `after_deploy` block.

**Review by hectcastro — APPROVED** [review]
👍

**Pull request merged**